.github/workflows/sycl_nightly.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       build_github_cache: true
       build_cache_root: "/__w/"
       build_artifact_suffix: default
+      lts_config: "ocl_gen9;ocl_x64;hip_amdgpu"
```

Comment:
Why `l0_gen9` is missing?

Reply:
I see the answer to this question in the description.

Reply:
I'm surprised because buildbot/sycl-ubu-x64-pr doesn't seem to have any test failures.

Reply:
> I'm surprised because buildbot/sycl-ubu-x64-pr doesn't seem to have any test failures.

Well, it doesn't run any tests on GPU...

Yes, I'm also surprised by the test failure, but since that is a simple FileCheck failure, it is very likely, that the test is just sensitive to the environment.

Reply:
Ah... it should Jenkins, not BuildBot and now I'm not surprised.